samples/snippets/data_client/data_client_snippets_async.py

```diff
@@ -42,24 +42,32 @@ async def write_batch(table):
     from google.cloud.bigtable.data import BigtableDataClientAsync
     from google.cloud.bigtable.data.mutations import SetCell
     from google.cloud.bigtable.data.mutations import RowMutationEntry
+    from google.cloud.bigtable.data.exceptions import MutationsExceptionGroup
 
     async def write_batch(project_id, instance_id, table_id):
         async with BigtableDataClientAsync(project=project_id) as client:
             async with client.get_table(instance_id, table_id) as table:
                 family_id = "stats_summary"
-
-                async with table.mutations_batcher() as batcher:
-                    mutation_list = [
-                        SetCell(family_id, "connected_cell", 1),
-                        SetCell(family_id, "connected_wifi", 1),
-                        SetCell(family_id, "os_build", "12155.0.0-rc1"),
-                    ]
-                    batcher.append(
-                        RowMutationEntry("tablet#a0b81f74#20190501", mutation_list)
-                    )
-                    batcher.append(
-                        RowMutationEntry("tablet#a0b81f74#20190502", mutation_list)
-                    )
+                try:
+                    async with table.mutations_batcher() as batcher:
+                        mutation_list = [
+                            SetCell(family_id, "connected_cell", 1),
+                            SetCell(family_id, "connected_wifi", 1),
+                            SetCell(family_id, "os_build", "12155.0.0-rc1"),
+                        ]
+                        await batcher.append(
```

Comment:
please add a comment that this await is for scheduling the mutation and not for applying the mutation...that will only happen when the context manager exits or batcher.close() is called